src/library/scala/util/Properties.scala

```diff
@@ -181,14 +203,19 @@ private[scala] trait PropertiesTrait {
    * }}}
    */
   def isJavaAtLeast(version: String): Boolean = {
-    def parts(x: String) = {
-      val i = x.indexOf('.')
-      if (i < 0) throw new NumberFormatException("Not a version: " + x)
-      (x.substring(0, i), x.substring(i+1, x.length))
```

Comment:
e.g. just ignore first part if present and equals 1 and second part < 9. Maybe rename `parts` to `specOf`...